frontend/packages/console-demo-plugin/src/plugin.tsx

```diff
@@ -167,6 +172,25 @@ const plugin: Plugin<ConsumedExtensions> = [
       render: () => <h1>Test Page</h1>,
     },
   },
+  {
+    type: 'Dashboards/Overview/Inventory/Item',
+    properties: {
+      resource: {
+        isList: true,
+        kind: RouteModel.kind,
+        prop: 'routes',
+      },
+      model: RouteModel,
+      filter: getRouteStatusGroups,
+    },
+  },
+  {
+    type: 'Dashboards/Inventory/Item/Group',
+    properties: {
+      id: 'demo-inventory-group',
```

Comment:
This doesn't match `demo-inventory-status` above in `getRouteStatusGroups` (?)

Reply:
oops, sorry about that. It got out-of-sync while doing previous changes

Reply:
fixed now